### PR TITLE
Update NUMPY_COMPARISON.md

### PR DESCRIPTION
**Pull request by esquerbatua**

### CODE REVIEW

File: docs/NUMPY_COMPARISON.md [hunk 1]
```diff
@@ -297,12 +297,13 @@ This is a collection of common exercises using numpy, and their equivalent solut
 ```
 
 ```v
->>> import vtl.num
+>>> import vtl.num as np
```
Comment (ulises-jeremias) [inline docs/NUMPY_COMPARISON.md:300]: why do we want to do this instead of using it as `num.{function_name}`?

**Comment by ulises-jeremias** [comment]
Closing this PR since the comments were not answered and this changes will be no needed ater we merge `feature/tensor`

**Comment by esquerbatua** [comment]
Sorry I was busy these days

File: docs/NUMPY_COMPARISON.md (hunk 1 above)
Comment (esquerbatua) [reply docs/NUMPY_COMPARISON.md:300]: Only to align it with the whole context and the title of the example: "1. Import the numpy package under the name np"

**Pull request closed without merging**